packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -10,7 +10,8 @@ import { ContainerWarning } from '@fluidframework/container-definitions/internal
 import { CreateChildSummarizerNodeFn } from '@fluidframework/runtime-definitions/internal';
 import { CreateChildSummarizerNodeParam } from '@fluidframework/runtime-definitions/internal';
 import { FluidDataStoreRegistryEntry } from '@fluidframework/runtime-definitions/internal';
-import { FluidObject } from '@fluidframework/core-interfaces';
+import { FluidObject } from '@fluidframework/core-interfaces/internal';
+import { FluidObject as FluidObject_2 } from '@fluidframework/core-interfaces';
```

Comment:
a few more import issues here

Reply:
This is not specific to this change. Its an interaction between how we do our release tagging and API extractor. It also shows up in https://github.com/microsoft/FluidFramework/pull/20554#discussion_r1575730978 and there is an API extractor bug tracking it. Its possible to make code changes to avoid this, but as it only impacts the imports in API reports, and the actual code that needs changing to fix it is not just in this PR but some existing stuff as well, I think we shouldn't worry about this here.


If we do want to adopt the policy that we are not allowed to import public types from internal rollups, then we should stop including them in the internal rollup instead of waiting for API reports to show issues like then then tracking them down.